x-pack/plugins/lists/common/schemas/types/entry_exists.test.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { pipe } from 'fp-ts/lib/pipeable';
+import { left } from 'fp-ts/lib/Either';
+
+import { foldLeftRight, getPaths } from '../../siem_common_deps';
```

Comment:
Try to pull from `shared_imports` when you can, I'd like to get rid of that file eventually.

Reply:
Updated and removed `siem_common_deps` in https://github.com/elastic/kibana/pull/73865